src/web/components/SharingPermission/BulkAddPermissions.scss

```diff
@@ -0,0 +1,53 @@
+.bulk-add-permissions {
+  display: flex;
+  flex-direction: column;
+  border: 2px solid var(--theme-action);
+  border-radius: 5px;
+  line-height: 1;
+  box-shadow: 0px 4px 14px var(--theme-box-shadow);
+
+  .bulk-add-permissions-header {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: 0 20px;
+    flex-direction: row;
+
+    h2 {
+      margin-top: 20px;
+    }
+
+    .recommendation-label {
+      background: var(--theme-label-background);
+      border-radius: 3px;
+      padding: 4px 8px;
+      margin-right: 8px;
+      display: flex;
+      align-items: center;
+      font-size: 0.75rem;
+      width: 115px;
```

Comment:
Setting an explicit width is probably going to be a bit flakey - it looks like it sizes basically the same without this.

Reply:
✔ removed